man/dcfldd.txt

```diff
@@ -97,6 +97,8 @@ OPTIONS
   vf=FILE                  Verify that FILE matches the specified input.
   verifylog=FILE           Send verify results to FILE instead of stderr.
   verifylog:=COMMAND       Exec and write verify results to process COMMAND.
+  diffwr=[on|off]          Write only to output if destination block content is differs. This operation 
+                           applies to any of=FILE that follows (diffwr= must be put before of=)
```

Comment:
The first sentence would be clear more like the one on `src/dcfldd.c` (see below for comments).